### PR TITLE
Test for Btrfs subvolumes using storage-ng

### PR DESCRIPTION
**Pull request by imobachgs**
Do not merge until storage-ng ISO is updated.

### CODE REVIEW

File: aytests/btrfs_copy_on_write_subvolumes.sh [hunk 1]
```diff
@@ -5,13 +5,13 @@
 
 set -e -x
 
-# search for the last '-' in "----------------- /" (copy-on-write enabled by default)
-lsattr -d / | grep -q -- "- /"
+# search for no flags '---' (copy-on-write enabled by default)
```
Comment (imobachgs) [inline aytests/btrfs_copy_on_write_subvolumes.sh:8]: Format has changed. Using `lsattr -dl` looks like a more robust way to check for flags:

```
/var/lib/pgsql               No_COW
```

**Review by ancorgs — APPROVED** [review]
LGTM

**Comment by imobachgs** [comment]
Test is outdated and superseded by new ones.

**Comment by imobachgs** [comment]
This PR is already outdated, so I am closing it.

**Pull request closed without merging**